tools/Config.mk

```diff
@@ -578,8 +578,8 @@ $(1)_$(2):
 endef
 
 export DEFINE_PREFIX := $(subst X,,${shell $(DEFINE) "$(CC)" "X"})
-export INCDIR_PREFIX := $(subst "X",,${shell $(INCDIR) "$(CC)" "X"})
-export INCSYSDIR_PREFIX := $(subst "X",,${shell $(INCDIR) -s "$(CC)" "X"})
+export INCDIR_PREFIX := $(subst "X",,${shell $(INCDIR) "$(CC)" "X" 2>/dev/null})
```

Comment:
/dev/null doesn't exist on Windows, it's better to check $(INCDIR) exist before invoke it

Reply:
Done～